### DISCUSSION TITLE
It look like I found new URL but it does not work with Invoke-WebRequest

### DISCUSSION THREAD

**Discussion by BrandonWanHuanSheng** [microsoft/winget-pkgs#108066] [Q&A] [answered]
It look like this URL is working on the browser.  https://help.viber.com/hc/en-us/ But Invoke-WebRequest and Wget give me forbidden error which is incorrect. 😞

**Comment by mdanish-kh** [comment, marked as answer]
Some websites block or limit access from non-browser clients or automated tools for various reasons. There isn't much we can do about it from WinGet's side, and the solution is to just exclude the URL from the manifest submission.
